spec/factories/requests.rb

```diff
@@ -28,29 +28,46 @@ def random_request_items
     request_items { random_request_items }
     comments { "Urgent" }
     partner_user { ::User.partner_users.first || create(:partners_user) }
-  end
 
-  trait :started do
-    status { 'started' }
-  end
+    # For compatibility we can take in a list of request_items and turn it into a
+    # list of item_requests
+    after(:build) do |request|
+      if request.item_requests.empty?
+        request.request_items.each do |request_item|
+          item = Item.find(request_item['item_id'])
+          request.item_requests << Partners::ItemRequest.new(
+            item_id: item.id,
+            quantity: request_item['quantity'],
+            name: item.name,
+            partner_key: item.partner_key,
+            request_unit: request_item["request_unit"]
+          )
+        end
+      end
+    end
 
-  trait :fulfilled do
```

Comment:
I'm not sure these ever worked since they weren't nested inside of the `factory` block.

Reply:
If they didn't work, can we just delete them?

Reply:
Sorry they WORK but they were being defined globally instead of scoped to only the Requests factory. So any other factory could have been using these traits on accident, though lucky us nothing else is using them. I verified that all of them ARE being used.